tasks/44_time-simulation/time-simulation.js

```diff
@@ -0,0 +1,23 @@
+class SimTime {
```

Comment:
Please see task description

function SimTime() {
  // ...
}

let sim = new SimTime();

Reply:
fixed it